src/pdfboxing/util.clj

```diff
@@ -0,0 +1,38 @@
+(ns pdfboxing.util)
+
+(defn first-line-valid?
```

Comment:
Thanks for the feedback.

I'm curious, how did you come across this project? 
What do you use it for? 

Thanks